### ISSUE TITLE
Choose two raster data sets for use by the API

### ISSUE THREAD

**Issue by kellyi** [azavea/geotrellis-collections-api-research#4]
Pick two raster data sets to use for the project & learn more about options for processing and storing them. 

For the project, these will likely be stored locally somewhere accessible to the Docker container, but this will be an occasion to learn more about:

* how raster data sets are processed to prepare them for use in our applications
* how & where these data sets are or can be stored for deployment apps

Deliverables for this are:

* pick and make available two data sets, with any necessary processing/ingest
* update to the project configuration to make use of the data sets & possibly something like a ping to confirm that they're available to the API
* update the README to explain how to set up a development environment using the data
* potentially add a Rake task to automate any required processing and setup steps

**Comment by kellyi** [comment]
For reference http://geotrellis.readthedocs.io/en/latest/guide/tile-backends.html

Started out on the path of using the file system, but may reconsider and just consume from S3.

**Comment by kellyi** [comment]
More reference: here's the set of rasters used in MMW:

https://github.com/WikiWatershed/model-my-watershed/blob/f4c6a4c0cdde099f0fb4ce82723e4c03ad53fce5/src/mmw/mmw/settings/base.py#L387

**Comment by kellyi** [comment]
Tentatively going to use 30m nlcd + slope:

```
nlcd-2011-30m-epsg5070-0.10.0
us-percent-slope-30m-epsg5070
```

...since they're both available in the datahub and could potentially merit interesting visualizations.

**Comment by kellyi** [comment]
Since those are also in 5070, will have to reproject the 3857 sent from the client. I assume it's faster to do this in GeoTrellis, but I may see what it'd look like to do it client-side.

https://github.com/perliedman/reproject

**Comment by kellyi** [comment]
> ..., but I may see what it'd look like to do it client-side.

This turned out not to be super-pleasant, so I'll do it API side instead.

**Comment by kellyi** [comment]
This was completed in:

11256abbaa3409c44205733ce718be886f9628db
2a703871e14174d9ff8e072b36cd8f982c929fe8

&

8ec4ab94f4f2becca7b8db69d1e3d944554c3e15

Rather than reading from a local file, I've set up the API to use credentials set via an env var on host to query the datahub. I also adjusted the README with some terse documentation on getting the data set up, and I added some catalog methods to the API endpoint to create an S3LayerReader GeoTrellis object. There's also now a `/ping` endpoint to call `catalog` & confirm it's working correctly.
